contracts/v0.8/utils/BigInts.sol

```diff
@@ -72,11 +73,15 @@ library BigInts {
     /// @return a int256 value and flog that indicates whether it was possible to convert or not (the value overflows int256 type)
     function toInt256(CommonTypes.BigInt memory value) internal view returns (int256, bool) {
         BigNumber memory max = BigNumbers.init(MAX_INT, false);
+        BigNumber memory min = BigNumbers.init(MIN_INT, true);
         BigNumber memory bigNumValue = BigNumbers.init(value.val, false);
```

Comment:
Since this `bigNumValue` is always initialized as a positive number, the checks for the lower boundary don't do anything.
Should we pass the `CommonTypes.BigInt.neg` as the param to `BigNumbers.init`?

```solidity
BigNumber memory min = BigNumbers.init(MIN_INT, value.neg);
```

Reply:
my mistake - added value.neg instead of false